### PR TITLE
Additional real world kivy language import examples

### PR DESCRIPTION
**Pull request by jtownley**
Goal: 
  Is to prevent people (like me) getting confused by the import syntax by adding 2 real world common examples to this process.

### CODE REVIEW

**Comment by dessant** [comment]
Nice addition, although you could merge it with the original comparison, like this:

```
To import something from python::

    #:import name x.y.z
    #:import isdir os.path.isdir
    #:import np numpy

Is equivalent to::

    from x.y import z as name
    from os.path import isdir
    import numpy as np
```

**Comment by jtownley** [comment]
I thought about that, but was worried about mixing theory with practice. Though I admit to being on the fence.

**Comment by inclement** [comment]
import os.path.isdir

I don't think this is actually valid python, you can only import modules this way.

**Comment by jtownley** [comment]
You are of course completely correct.

File: doc/sources/guide/lang.rst [hunk 1]
```diff
@@ -77,13 +77,19 @@ There are two special syntax to define values for the whole Kv context:
 To import something from python::
 
     #:import name x.y.z
+    #:import isdir os.path.isdir
+    #:import np numpy
+
```
Comment (dessant) [inline doc/sources/guide/lang.rst]: Please remove this newline and we can merge. ;)

File: doc/sources/guide/lang.rst [hunk 2]
```diff
@@ -77,13 +77,19 @@ There are two special syntax to define values for the whole Kv context:
...
 
 Is equivalent to::
 
     from x.y import z as name
+    from os.path import isdir
+    import numpy as np
 
 in python.
 
+
```
Comment (dessant) [inline doc/sources/guide/lang.rst]: same

**Comment by dessant** [comment]
Thanks a bunch. :+1:

**Pull request merged**